starknet-rpc-test/get_block_transaction_count.rs

```diff
@@ -0,0 +1,101 @@
+#![feature(assert_matches)]
+
+extern crate starknet_rpc_test;
+
+use std::assert_matches::assert_matches;
+
+use rstest::{fixture, rstest};
+use starknet_accounts::SingleOwnerAccount;
+use starknet_core::chain_id;
+use starknet_core::types::{BlockId, BlockTag};
+use starknet_ff::FieldElement;
+use starknet_providers::{Provider, ProviderError};
+use starknet_rpc_test::constants::{ARGENT_CONTRACT_ADDRESS, MINT_AMOUNT, SIGNER_PRIVATE};
+use starknet_rpc_test::utils::AccountActions;
+use starknet_rpc_test::{ExecutionStrategy, MadaraClient};
+use starknet_signers::{LocalWallet, SigningKey};
+
+#[fixture]
+async fn madara() -> MadaraClient {
+    MadaraClient::new(ExecutionStrategy::Native).await
+}
```

Comment:
This will run a new node for each test. Is it the behavior you wanted?
Otherwise, you can use `#[once]`
https://docs.rs/rstest/latest/rstest/attr.fixture.html#once-fixture

Reply:
Not usable for async fixtures, trying to find another solution..